vignettes/getting-started.Rmd

```diff
@@ -34,9 +37,11 @@ colnames(example_data)
 
 ## Setting up data
 
-We're going to set up three datasets. The first is required, its the counts that the genetic interaction analysis will be used for. 
+We're going to set up three datasets. The first is required, it's the counts that the genetic interaction analysis will be used for. 
 
 ```{r}
+# TODO: ## perhaps here we can clarify the flexibility of the workflow / what can be set up. e.g. does this function allow exploration / comparison of specific days and reps? or is that not really needed for the purposes of the vignette?
```

Comment:
That will happen later on in this vignette, but we should give a preview that if comparisons like that need to be made then someone should have proper labels in their metadata to make those comparisons. Would you be able to add this kind of description @danieljgroso ?